pkgs/development/python-modules/pygetwindow/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyrect
+}:
+buildPythonPackage rec {
+  pname = "PyGetWindow";
+  version = "0.0.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-F4lDVefSswXNgy1xdwg4QBfBaYqQziT29/vwJC3Qpog=";
+  };
+
+  doCheck = false;
```

The project has a test directory.

---

This lib is required by pyautogui but doesn't support Linux, it supports windows fully and MacOS partially.

The tests would fail as the imports also fail.

Should I just remove it from PyAutoGUI setup.py?

---

Seems like the objective of the project is to eventually support Linux: https://github.com/asweigart/PyGetWindow/blob/master/src/pygetwindow/__init__.py#L347-L349.

So I would keep it. Besides, we also needs to support macOS.

---

Maybe just add a note about this this
```suggestion
  # Doesn't support Linux currently and partial support for Darwin (making tests fail)
  doCheck = false;
```